py/selenium/webdriver/edge/options.py

```diff
@@ -16,21 +16,31 @@
 # under the License.
 
 from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
-from selenium.webdriver.common.options import BaseOptions
+from selenium.webdriver.chromium.options import ChromiumOptions
 
 
-class Options(BaseOptions):
+class Options(ChromiumOptions):
+    KEY = "goog:chromeOptions"
 
-    def __init__(self):
-        super(Options, self).__init__()
-        self._page_load_strategy = "normal"
+    def __init__(self, is_legacy=True):
+        super(Options, self).__init__(self.KEY if not is_legacy else "")
+        self._is_legacy = is_legacy
+
+        if is_legacy:
+            self._page_load_strategy = "normal"
 
     @property
     def page_load_strategy(self):
+        if not self._is_legacy:
+            raise AttributeError("Page Load Strategy does not exist only in Legacy Mode")
```

Comment:
I can't parse the meaning of this error message.  can you change the wording to something more direct, like:
"Page Load Strategy only exists in Legacy Mode"

Reply:
Sorry, you're right. I meant to fix that but forgot. Thanks